dist/quick-lint-js.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs, stdenv ? pkgs.stdenv, lib ? stdenv.lib, mkDerivation ? stdenv.mkDerivation }:
+
+mkDerivation rec {
+  pname = "quick-lint-js";
+  version = "master";
+
+  src = ../.;
+  unpackPhase = null;
+
+  buildInputs = with pkgs; [ cmake ninja ];
+  cmakeBuildType = "Release";  # Or Debug, depending on what you want
```

Comment:
You are correct - defining `Release` is optional since it defaults to it, but I used this to make it very easy for the next person to find this option. Since, looking at your other comments, I'll have to introduce a way to offer easily switching between `Release` and `Debug` anyway, this line will get redone and might not be necessary.